<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftwareFontPermitDlg</class>
 <widget class="BaseDlg" name="SoftwareFontPermitDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SoftwareFontPermitDlg</string>
  </property>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>745</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>740</width>
     <height>618</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#textBrowser QScrollBar::handle:vertical  {
    margin: 2px;
	width: 9px;
}

#textBrowser QScrollBar:vertical  {
    border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
	width: 13px;
}

#textBrowser QScrollBar::sub-line:vertical  {
    height: 0; 
    border-bottom: 0;
}

QTextBrowser  {
	padding-right: 0;
    border-radius: 0;
}</string>
   </property>
   <property name="html">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>SoftwareFontPermitDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>597</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
